f5/bigip/__init__.py

```diff
@@ -71,10 +71,6 @@ def hostname(self):
     def icontrol_version(self):
         return self._meta_data['icontrol_version']
 
-    @property
```

Comment:
Backwards compat was broken with this change. I should have recognized it and we should have release 2.0.0.